src/main/java/cz/xtf/openshift/OpenShiftUtil.java

```diff
@@ -523,6 +532,10 @@ public RoleBinding getRoleBinding(String name) {
 		return client.roleBindings().list().getItems();
 	}
 
+	public List<RoleBinding> getUserRoleBindings() {
```

Comment:
needs javadoc to describe what the "user" role bindings are supposed to mean

Reply:
extended with javadoc